javaharness/java/arcs/android/client/ArcsServiceBridge.java

```diff
@@ -71,13 +70,12 @@ public void registerRenderer(String modality, IRemoteOutputCallback callback) {
     runServiceMethod(service -> service.registerRenderer(modality, callback));
   }
 
-  @Override
   public void sendMessageToArcs(String message) {
     runServiceMethod(service -> service.sendMessageToArcs(message));
   }
 
   /**
-   * Connects to the Arcs service (if not already connected), and then runs the given service
+   * Connects to the Constants service (if not already connected), and then runs the given service
```

Comment:
same question - what is "Constants service"?
all i see with the name `Constants` is an interface with constants only, but it appears a lot in comments and logs, now sure i understand.
Also I don't think this is a good name - "Arcs API" and "Arcs service" seems clearer to me.

Reply:
A common Java idiom is to make an interface with a bunch of constants, and then "implement" it to import them into your class, perhaps that's what it is.